src/components/contact/contactFAQ/ContactFAQ.tsx

```diff
@@ -0,0 +1,134 @@
+import React, { FC } from "react";
+
+import { AccordionBox, StyledAccordion, PanelBox } from "./ContactFAQ.css";
+import { TitleText } from "../contactCTA/ContactCTA.css";
+
+import Accordion from "react-bootstrap/Accordion";
+
+const ContactFAQ: FC = () => {
+  return (
+    <PanelBox id="panels-box">
+      <TitleText>F.A.Q.</TitleText>
+      <AccordionBox id="accordion-box">
+        <StyledAccordion flush>
+          <Accordion.Item eventKey="0">
+            <Accordion.Header>
+              <b>How do I add a resource?</b>
+            </Accordion.Header>
+            <Accordion.Body>
+              Elit dolor irure dolore reprehenderit Lorem ad id culpa nulla
```

Comment:
Do we have copy from the mockups or others that can be substituted here. Let's get rid of the Latin.

Reply:
Yeah I agree, I used all the copy available in the mockups. Not sure where else to look..

Reply:
I think we can close this for now.

Reply:
For sure we need a check-in regarding the copy. The dang VOLUNTEER header for the contact us page is “VOLUNTEERING” for the mobile view. :/